yarn.lock

```diff
@@ -560,20 +560,28 @@
   dependencies:
     "@sinonjs/commons" "^1.7.0"
 
+"@sofie-automation/shared-lib@46.0.0":
+  version "1.47.0-nightly-release47-20221012-150626-b39067d.0"
```

Comment:
I am aware that this is not the version we want for our shared-lib.
We currently don't have our own version of shared-lib, so we need to use some version.
However, the actions fails while using this version, so we need to make some decision regards having a shared-lib library of our own

Reply:
We are now dependent on our shared-lib library